cli/src/pcluster/config/cluster_config.py

```diff
@@ -1188,8 +1198,8 @@ def partition(self):
         return get_partition()
 
     @property
-    def mount_dir_list(self):
-        """Retrieve the list of mount dirs for the shared storage and head node ephemeral volume."""
+    def shared_mount_dir_list(self):
+        """Retrieve the list of shared mount dirs for the shared storage and head node ephemeral volume."""
```

Comment:
the comment is wrong, it is not returning the head node ephemeral volume.

Reply:
Done